libs/utils/src/lib/serializer.service.ts

```diff
@@ -114,15 +115,19 @@ export class Serializer {
   }
 
   private completionSchematicType(collection: string, name: string): any {
-    if (collection === '@nrwl/schematics' && name === 'module') {
-      return 'absoluteModules';
-    }
-    if (
-      collection === '@schematics/angular' ||
-      collection === '@ngrx/schematics'
-    ) {
-      if (name === 'project') return 'projects';
-      if (name === 'module') return 'localModules';
+    switch (collection) {
```

Comment:
As the list of collections is growing I thought a switch was a bit more readable here.

Reply:
Definitely!